File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        length = len(nums)
        hashmap = {}
        result = []
        for num in nums:
            if num in hashmap:
                hashmap[num] +=1
            else:
                hashmap[num] = 1
        sorted_value = sorted(hashmap.items(), key=lambda x:x[1], reverse=True)
        for i in range(k):
            result.append(sorted_value[i][0])

        return result
        

        

File: Data Structures & Algorithms/two-integer-sum/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        if(nums.length === 0 ) {
            return 
        }

        const result = [];
        for (let i = 0; i<nums.length-1; i++) {
            for (let j=i+1; j<nums.length; j++) {
                if(nums[i]+nums[j] === target) {
                    result.push(i,j);
                }
            }
        }
        return result;
    }
}
